### PR TITLE
Add skyscanner api

### PR DESCRIPTION
**Pull request by ThisIsMatin**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1157,6 +1157,7 @@ API | Description | Auth | HTTPS | CORS |
 | [OpenSky Network](https://opensky-network.org/apidoc/index.html) | Free real-time ADS-B aviation data | No | Yes | Unknown |
 | [REFUGE Restrooms](https://www.refugerestrooms.org/api/docs/#!/restrooms) | Provides safe restroom access for transgender, intersex and gender nonconforming individuals | No | Yes | Unknown |
 | [Schiphol Airport](https://developer.schiphol.nl/) | Schiphol | `apiKey` | Yes | Unknown |
+| [Skyscanner](https://rapidapi.com/skyscanner/api/skyscanner-flight-search) | The Skyscanner API lets you search for flights & get flight prices from Skyscanner’s database of prices, as well as get live quotes directly from ticketing agencies. | `apiKey` | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:1160]: description should not end with .
description should not exceed 100 characters (currently 165)

Fix this, please

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @ThisIsMatin ;)

Can you resolve this review, please?

**Comment by ThisIsMatin** [comment]
Sorry for the inconvenience, the issue has been resolved :)

**Comment by ThisIsMatin** [comment]
Sorry for the inconvenience @matheusfelipeog , the issue has been resolved :)

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ✔

Thanks for your update <3

**Pull request merged**